### PR TITLE
Adds 4 Hoppy feeds in BE to systems.csv

### PR DESCRIPTION
**Pull request by richfab**
https://behoppy.eu/

### CODE REVIEW

**Review by josee-sabourin — APPROVED** [review]

File: systems.csv [hunk 1]
```diff
@@ -44,6 +44,10 @@ BE,Dott Charleroi,"Charleroi,BE",dott-charleroi,https://ridedott.com,https://gbf
 BE,Dott Ghent,"Ghent,BE",dott-ghent,https://ridedott.com,https://gbfs.api.ridedott.com/public/v2/ghent/gbfs.json,
 BE,Dott Liege,"Liege,BE",dott-liege,https://ridedott.com,https://gbfs.api.ridedott.com/public/v2/liege/gbfs.json,
 BE,Dott Namur,"Namur,BE",dott-namur,https://ridedott.com,https://gbfs.api.ridedott.com/public/v2/namur/gbfs.json,
+BE,Hoppy,"Ghent, BE",hoppsharing_gent,https://behoppy.eu/,https://eu-mobility.joyride.tech/api/v1/gbfs/a640fd4b-5a16-48c6-9aa2-d72674caf5f8/3e98a50a40a54d4884c6aa01d8c18510/gbfs.json
+BE,Hoppy,"Hasselt, BE",hoppy_Hasselt,https://behoppy.eu/,https://eu-mobility.joyride.tech/api/v1/gbfs/a640fd4b-5a16-48c6-9aa2-d72674caf5f8/2a29c4c627e14399a19436855d25ceb9/gbfs.json
+BE,Hoppy,"Mechelen, BE",hoppy_Mechelen,https://behoppy.eu/,https://eu-mobility.joyride.tech/api/v1/gbfs/a640fd4b-5a16-48c6-9aa2-d72674caf5f8/1c03f680d52141df85c33675bea6d625/gbfs.json
+BE,Hoppy,"Blankenberge, BE",hoppy_Blankenberge,https://behoppy.eu/,https://eu-mobility.joyride.tech/api/v1/gbfs/a640fd4b-5a16-48c6-9aa2-d72674caf5f8/57cb68a183234d0189474e7122944d81/gbfs.json
```
Comment (davidgamez) [inline systems.csv]: We are missing here an _empty_ column for _Authentication Info URL_ 
```suggestion
BE,Hoppy,"Ghent, BE",hoppsharing_gent,https://behoppy.eu/,https://eu-mobility.joyride.tech/api/v1/gbfs/a640fd4b-5a16-48c6-9aa2-d72674caf5f8/3e98a50a40a54d4884c6aa01d8c18510/gbfs.json,
BE,Hoppy,"Hasselt, BE",hoppy_Hasselt,https://behoppy.eu/,https://eu-mobility.joyride.tech/api/v1/gbfs/a640fd4b-5a16-48c6-9aa2-d72674caf5f8/2a29c4c627e14399a19436855d25ceb9/gbfs.json,
BE,Hoppy,"Mechelen, BE",hoppy_Mechelen,https://behoppy.eu/,https://eu-mobility.joyride.tech/api/v1/gbfs/a640fd4b-5a16-48c6-9aa2-d72674caf5f8/1c03f680d52141df85c33675bea6d625/gbfs.json,
BE,Hoppy,"Blankenberge, BE",hoppy_Blankenberge,https://behoppy.eu/,https://eu-mobility.joyride.tech/api/v1/gbfs/a640fd4b-5a16-48c6-9aa2-d72674caf5f8/57cb68a183234d0189474e7122944d81/gbfs.json,
```

Comment (richfab) [reply systems.csv]: Thank you for catching this @davidgamez 💜

**Comment by futuretap** [comment]
Thanks for publishing the feeds @ Hoppy. Can they please fix the deep links in `free_bike_status.json`? At least the iOS links don’t work. The Hoppy app is not registered as a Universal Link for the `eu-mobility.joyride.tech` domain.

Also if Hoppy could add range information as alerted by the GBFS validator, that would be terrific! (Also, feeds for Knokke and Rhodos!)

**Comment by josee-sabourin** [comment]
Thanks for this feedback @futuretap, I will reach out to Hoppy.

**Pull request merged**